File: Backend/Dtos/GameDto.cs
namespace Backend.Dtos
{
    public record class GameDto(
        string GameId,
        string Title,
        string ReleaseDate,
        string AboutGame,
        string ShortDescription,
        string Description,
        decimal Price,
        string HeaderImage,
        int MetacriticScore,
        int PositiveRatings,
        int NegativeRatings,
        List<string> Developers,
        List<string> Publishers,
        List<string> Categories,
        List<string> Genres,
        List<string> Screenshots,
        List<string> Tags
    );
}

File: Frontend/src/app/game/dto/game.dto.ts
export interface GameDto {
  gameId: string;
  title: string;
  releaseDate: string;
  aboutGame: string;
  shortDescription: string;
  description: string;
  price: number;
  headerImage: string;
  metacriticScore: number;
  positiveRatings: number;
  negativeRatings: number;

  developers?: string[];
  publishers?: string[];
  categories?: string[];
  genres?: string[];
  screenshots?: string[];
  tags?: string[];
}
